src/main/java/frc/robot/Robot.kt

```diff
@@ -61,6 +63,10 @@ object Robot : LoggedRobot() {
         driveController.rightBumper().whileTrue(Chassis.driveToRightBranch)
         driveController.y().whileTrue(Elevator.manualUp)
         driveController.a().whileTrue(Elevator.manualDown)
+        driveController
+            .start()
+            .onTrue(Commands.runOnce({ Chassis.resetRotation(Chassis.operatorForwardDirection) }))
+        SmartDashboard.putData("Zero wrist", Commands.runOnce({ Wrist.resetPosition(0.rotations) }))
```

Comment:
Make these commands in their respective classes with named function / variable, then it's good to go